### PR TITLE
Added link to new react native support policy to RNW support docs

### LINKED ISSUE #734: Add link to RN core support policy to RNW support policy doc
### Page url

https://microsoft.github.io/react-native-windows/support

### Problem Description

RN has their new support policy doc here: https://github.com/reactwg/react-native-releases

We should link to it within our policy doc.

### PR DESCRIPTION
**Pull request by YajurG**
## Description

Added link to new RN support policy within our RNW support documentation.

### Why

Resolves #734 

## Screenshots
image

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/742)

### CODE REVIEW

File: website/pages/en/support.js [hunk 1]
```diff
@@ -125,6 +125,11 @@ An *Unsupported* stable version of RNW should not expect any further updates, ch
 The RNW Team does not commit to any support for these releases.
 
 Customers are encouraged to upgrade away from *Unsupported* versions as soon as possible.
+
+### React Native Support Policy
+
+Linked is the new official React Native [Support Policy](https://github.com/reactwg/react-native-releases).
```
Comment (TatianaKapos) [inline website/pages/en/support.js:131]: nit (short for nit-picking): I would remove the word "new" so you won't have to go in and change it when it's not new anymore.

**Review by TatianaKapos — APPROVED** [review]

File: website/pages/en/support.js (hunk 1 above)
Comment (chrisglein) [inline website/pages/en/support.js:131]:
```suggestion
Linked is the official React Native [Support Policy](https://github.com/reactwg/react-native-releases).
```

Comment (jonthysell) [inline website/pages/en/support.js:131]:
```suggestion
For the official support policy of React Native releases, see: [React Native Support Policy](https://github.com/reactwg/react-native-releases).
```

**Review by jonthysell — APPROVED** [review]
Minor rewording, but looks good. :)

File: website/pages/en/support.js (hunk 1 above)
Comment (jonthysell) [reply website/pages/en/support.js:131]: Or perhaps more simply:

```suggestion
See the official [React Native Support Policy](https://github.com/reactwg/react-native-releases).
```

Comment (chrisglein) [reply website/pages/en/support.js:131]: Jon's suggestion is better :)

File: website/pages/en/support.js [hunk 2]
```diff
@@ -125,6 +125,11 @@ An *Unsupported* stable version of RNW should not expect any further updates, ch
 The RNW Team does not commit to any support for these releases.
 
 Customers are encouraged to upgrade away from *Unsupported* versions as soon as possible.
+
+### React Native Support Policy
```
Comment (jonthysell) [inline website/pages/en/support.js:129]:
```suggestion
## React Native Support Policy
```

File: website/pages/en/support.js (hunk 1 above)
Comment (YajurG) [reply website/pages/en/support.js]: Changed the wording to 'See the official...'

File: website/pages/en/support.js [hunk 3]
```diff
@@ -125,6 +125,11 @@ An *Unsupported* stable version of RNW should not expect any further updates, ch
 The RNW Team does not commit to any support for these releases.
 
 Customers are encouraged to upgrade away from *Unsupported* versions as soon as possible.
+
+## React Native Support Policy
+
+See the official React Native [Support Policy](https://github.com/reactwg/react-native-releases).
```
Comment (chrisglein) [inline website/pages/en/support.js:131]: @jonthysell Is there an aka.ms for the support policy? Or is this url fine?

**Review by chrisglein — APPROVED** [review]

**Pull request merged**